spec/views/spree/admin/payment_methods/index.html.haml_spec.rb

```diff
@@ -19,10 +21,52 @@ def object_url(object = nil, options = {}) "" end
   end
 
   describe "payment methods index page" do
-    it "shows only the providers of the existing payment methods" do
-      render
+    context "when user is not admin" do
+      before do
+        allow(view).to receive_messages spree_current_user: create_enterprise_user
```

Comment:
this PR needs rebasing. the helper is now AuthenticationHelper and create_enterprise_user is gone. It should be simply
create(:user)

Reply:
Done